test/org/zaproxy/zap/extension/ascanrules/ActiveScannerTest.java

```diff
@@ -20,9 +20,9 @@
 package org.zaproxy.zap.extension.ascanrules;
 
 import java.io.File;
-import java.io.FileInputStream;
```

Comment:
I don't see anything wrong with this PR, but the description doesn't match `org.testng.reporters.Files`